Cpm.py

```diff
@@ -4,43 +4,98 @@
 class Activity:
 
     def __init__(self, name, duration):
-        self._name = name
-        self._duration = duration
-        self._depending_activities = []
+        self.name = name
+        self.duration = duration
+        self.min_time_to_start = 0
+        self.max_time_to_start = 0
+        #self._depending_activities = []
 
     def __str__(self):
-        print("Activity %s with duration %s", self._name, self._duration)
+        print("Activity %s with duration %s", self.name, self.duration)
+
+    def __eq__(self, other):
+        return self.name == other.name and self.duration == other.duration
```

Comment:
not sure about it, maybe we should generate a UUID to each activity on create time and just compare it.